NSS-Day-Cohort-58/game-review-api
# Models and views

Completes ticket #4 

## Steps to test

1. `git fetch --all`
1. `git checkout models-and-views`
1. Verify that views and models directories exist
1. Verify that all 6 files mentioned in the ticket exist in those directories
